cypress/utils/modules/StaffPage/StaffPageTestFunctions.ts

```diff
@@ -0,0 +1,97 @@
+import { LanguageChangeTestFunctions } from '../../LanguageChange/LanguageChangeTestFunctions';
+
+const LanguageChange = new LanguageChangeTestFunctions();
+
+export class StaffPageTestFunctions {
+  // main panel
+  private getStaffTile = () => cy.getBySelector('link-to-staff');
+  private getStaffIcon = () => cy.getBySelector('people-alt-icon');
+  private getStaffLink = () => cy.getBySelector('link-to-staff-text');
+  private getStaffListTile = () => cy.getBySelector('details-tile');
+
+  private testStaffPageTile = () => {
+    this.getStaffTile().should('exist');
+    this.getStaffIcon().should('exist');
+  };
```

Comment:
this method should be declared in MainPanelTestFunctions since it tests MainPanel